src/commonMain/kotlin/fr/acinq/lightning/channel/Helpers.kt

```diff
@@ -339,12 +338,11 @@ object Helpers {
                 }
             }
 
-            val fundingPubKey = keyManager.fundingPublicKey(localParams.fundingKeyPath)
-            val channelKeyPath = keyManager.channelKeyPath(localParams, channelVersion)
-            val commitmentInput = makeFundingInputInfo(fundingTxHash, fundingTxOutputIndex, fundingAmount, fundingPubKey.publicKey, remoteParams.fundingPubKey)
-            val localPerCommitmentPoint = keyManager.commitmentPoint(channelKeyPath, 0)
-            val localCommitTx = Commitments.makeLocalTxs(keyManager, channelVersion, 0, localParams, remoteParams, commitmentInput, localPerCommitmentPoint, localSpec).first
-            val remoteCommitTx = Commitments.makeRemoteTxs(keyManager, channelVersion, 0, localParams, remoteParams, commitmentInput, remoteFirstPerCommitmentPoint, remoteSpec).first
+            val fundingPubKey = localParams.channelKeys.fundingPubKey
+            val commitmentInput = makeFundingInputInfo(fundingTxHash, fundingTxOutputIndex, fundingAmount, fundingPubKey, remoteParams.fundingPubKey)
+            val localPerCommitmentPoint = keyManager.commitmentPoint(localParams.channelKeys.shaSeed, 0)
+            val localCommitTx = Commitments.makeLocalTxs( 0, localParams, remoteParams, commitmentInput, localPerCommitmentPoint, localSpec).first
```

Comment:
nit: reformat this file (this unnecessary space is in multiple places):
```suggestion
            val localCommitTx = Commitments.makeLocalTxs(0, localParams, remoteParams, commitmentInput, localPerCommitmentPoint, localSpec).first
```

Reply:
yes I upgraded/downgraded idea many times and lost the "save actions" configurations :(
done in https://github.com/ACINQ/lightning-kmp/pull/255/commits/3b01af57bbb8bc95af625ce1d11ae54e433214de